interface/wx/arrstr.h

```diff
@@ -403,11 +407,98 @@ int wxDictionaryStringSortAscending(const wxString& s1, const wxString& s2);
 
     See wxDictionaryStringSortAscending() for the dictionary sort description.
 
-    @see wxStringSortDescending()
+    @see wxDictionaryStringSortAscending(),
+         wxStringSortDescending(),
+         wxNaturalStringSortDescending()
 
     @since 3.1.0
  */
-int wxDictionaryStringSortAscending(const wxString& s1, const wxString& s2);
+int wxDictionaryStringSortDescending(const wxString& s1, const wxString& s2);
+
+
+    /**
+        Comparison function used for Natural Sort.
+        
+        Functions in the same way as wxDictionaryStringSortAscending(), with
+        the exception that numbers within the string are recognised, and
+        compared numerically, rather than alphabetically. When used for
+        sorting, the result is that e.g. file names containing numbers are
+        sorted in a natural way.
+
+        This function will use an OS native function if one is available,
+        to ensure that the sort order is the same as the OS uses.
+        
+        Comparison is case insensitive.
+            
+        e.g. Sorting using wxDictionaryStringSortAscending() results in:
+        - file1.txt
+        - file10.txt
+        - file100.txt
+        - file2.txt
+        - file20.txt
+        - file3.txt
+        
+        e.g. Sorting using wxNaturalStringSortAscending() results in:
+        - file1.txt
+        - file2.txt
+        - file3.txt
+        - file11.txt
+        - file20.txt
+        - file100.txt
+        
+        @see wxNaturalStringSortDescending(),
+             wxStringSortAscending(),
+             wxDictionaryStringSortAscending()
+        
+        @since 3.1.2  
```

Comment:
```suggestion
        @since 3.1.4
```
(also elsewhere)

Reply:
As I wrote in the PR, I have not touched the docs yet, as I thought this PR had little chances to get accepted.